src/app/zwiazek/zarzad/page.tsx

```diff
@@ -1,7 +1,60 @@
+import { boardData } from "@/data/boardData";
+import LogoAK from "@/icons/LogoAK";
+import Container from "@/components/shared/Container";
+
 export default function BoardPage() {
   return (
-    <div>
-      <p>Zarząd</p>
-    </div>
+    <section className="flex flex-col justify-center">
+      <Container>
+        <section className="pt-5 tablet:pb-[64px] desktop:py-10">
+          {/* todo: replace to a proper Breadcrumbs component */}
+          <h3 className="font-lora text-28 text-greenMain" aria-label="Ścieżka nawigacji">
+            Breadbrumbs Placeholder
+          </h3>
+        </section>
+        <div className="flex justify-center py-10 tablet:hidden">
+          <LogoAK className="h-[64px] w-[176px]" aria-label="Logo AK" />
+        </div>
+      </Container>
+      <div className="desktop:flex desktop:flex-col desktop:items-center desktop:justify-center">
+        <article
+          className="font-16 h-fit w-full bg-greenMain py-6 leading-6 text-backgroundMain desktop:min-h-[740px] desktop:w-[1022px]"
+        >
+          <Container>
+            <div className="mobile:hidden tablet:block">
+              <div className="flex flex-col items-center px-[88px] pt-3">
+                <LogoAK className="h-[100px] w-[275px]" aria-label="Logo AK" />
+                <hr className="my-9 w-full" aria-hidden="true"></hr>
```

Comment:
I think this could be a potential candidate for a separate component (`Divider`) as it is used in few places as well.